src/angular-resizable.js

```diff
@@ -1,5 +1,5 @@
 angular.module('angularResizable', [])
-    .directive('resizable', function() {
+    .directive('resizable', function($parse) {

```

Comment:
Hi, consider an array literal here to include the '$parse' dependency as a string. Minifying the js can/will end up in the injector not being able to resolve the dependency. Thanks!